### ISSUE TITLE
JAWS does not announce role as 'Dialog' on IE browser.

### ISSUE THREAD

**Issue by micurs** [FreedomScientific/VFO-standards-support#365]
### Summary

It appears that in some cases JAWS does not announce a modal with `role="dialog"` on IE.

This was encountered on Linkedin.com. To reproduce you need a linkedin account
and you need to have created a group. 
Once you have those in your account here are the steps to reproduce:

1. Open URL "https://www.linkedin.com/" in browser and login via valid credentials.
2. Home page will open and then navigate to any Groups present on right side nav in which you are owner and activate it.
3. Group page will open and then navigate to 'Edit Group details' button and activate it.
4. Edit group dialog will open.
5. Verify whether JAWS announces role as 'Dialog' or not.

### Expected Result :

JAWS should announce role as 'Dialog' for 'Edit group' control on IE browser.

### Actual Result :

JAWS does not announce role as 'Dialog' for 'Edit group' control on IE browser.
 
#### Operating System and version

Windows 10

#### Browser and version:

IE 11

**Comment by aardrian** [comment]
Ran some tests today using JAWS 2026 / Chrome 147.

Visit https://cdpn.io/pen/debug/wvWKvbE

1. activate the button _Focus the dialog_, hear "frank heading level two"
2.  activate the button _Focus the heading_, hear "frank modal dialog, frank heading level two"
3.  activate the button _Focus the button_, hear "frank modal dialog, close button"

Visit https://cdpn.io/pen/debug/bNdWrjO

1. activate _Edit your address_ in the _Possibly Good?_ column, hear "close button"
2. activate _Log in_ in the _Possibly Good?_ column, hear "login modal dialog, login group, username edit"
3. activate _Edit your address_ in the _Probably Less Good?_ column, hear "address changes modal dialog, Current Shipping Address group, address"
4. activate _Log in_ in the _Probably Less Good?_ column, hear "close button"

TL;DR:
* Focusing a dialog does not expose its role.
* Focusing something _in_ the dialog exposes the dialog role.
